config/core/test-configs.yaml

```diff
@@ -2276,6 +2286,7 @@ test_configs:
   - device_type: panda
     test_plans:
       - baseline
+      - kselftest-lkdtm
```

Comment:
I think the `panda` board is not really well supported any more, there's only one in Collabora's lab and it's offline. So it's probably best to drop this.